geode-core/src/main/java/org/apache/geode/management/internal/beans/MemberMBean.java

```diff
@@ -148,7 +148,12 @@ public long getInitialImageTime() {
 
   @Override
   public int getInitialImagesInProgres() {
```

Comment:
This is our internal implementation of `GEODE/geode-core/src/main/java/org/apache/geode/management/MemberMXBean.java`.  I was hesitant to remove the typo'd version from the public-facing interface without at least one version with it `@Deprecated`.